client/state/ui/comments/selected/reducer.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { has, keyBy, map, omit, partialRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_CLEAR_SELECTED,
+	COMMENTS_SELECT_ALL,
+	COMMENTS_TOGGLE_SELECTED,
+} from 'state/action-types';
+import { keyedReducer } from 'state/utils';
+
+const mapComments = partialRight( map, ( { commentId, postId } ) => ( { commentId, postId } ) );
```

Comment:
Do we really need `partialRight` here?
Wouldn't it be more readable by simply using `map` below?